src/MICore/Debugger.cs

```diff
@@ -971,19 +971,34 @@ internal void OnComplete(Results results, MICommandFactory commandFactory)
 
             internal void Abort()
             {
-                _completionSource.SetException(new ObjectDisposedException("Debugger"));
+                _completionSource.SetException(new DebuggerDisposedException(Command));
             }
         }
 
         private readonly Dictionary<uint, WaitingOperationDescriptor> _waitingOperations = new Dictionary<uint, WaitingOperationDescriptor>();
 
+        /// <summary>
+        /// Returns true it is a (gdb) prompt.
+        /// </summary>
+        /// <param name="prompt">The prompt from the debugger</param>
+        /// <returns>True if (gdb) prompt, false otherwise</returns>
+        /// <remarks>For SSH transport connecting to gdb, the (gdb) prompt string as a following space following it.</remarks>
```

Comment:
You might want to fix this sentence for grammar. (From what I can tell you are saying (gdb) prompt has a space following it)